en/security/index.md

```diff
@@ -19,7 +19,7 @@ will be published after fixes.
 If you have found an issue affecting one of our websites, please
 report it [via GitHub](https://github.com/ruby/www.ruby-lang.org/issues/new) or you can check our [Google Groups](https://groups.google.com/g/ruby-security-ann) for security announcements.
 
-If you have found an issue that affects a specific Ruby gem, follow the
+If you have found an issue that affects a specific Ruby community’s gem, follow the
```

Comment:
I'm not sure this improves the content, actually I do not fully understand what the intended meaning is.

Also, it should be apparent from the commit message / PR title, that the `en` page is changed; or rather, changes to `en` should be submitted in separate PRs.